resources/views/home.blade.php

```diff
@@ -18,6 +18,45 @@
                 </div>
             </div>
         </div>
+        <div class="col-md-8">
+          <br>
+          <form method="post" action="{{url('add_post')}}" enctype="multipart/form-data">
```

Comment:
There is no need to use enctype attribute on this form; this is only used when the form supports file attachments.

Reply:
Thanks, enctype attribute is now remove